kiwi/container/oci.py

```diff
@@ -204,17 +204,13 @@ def create(self, filename, base_image):
         Command.run(
             [
                 'umoci', 'config'
-            ] +
-            self.maintainer +
-            self.user +
-            self.workingdir +
-            self.entry_command +
-            self.entry_subcommand +
-            self.expose_ports +
-            self.volumes +
-            self.environment +
-            self.labels +
-            [
+            ] + list(
+                self.maintainer + self.user + self.workingdir
+            ) + list(
+                self.entry_command + self.entry_subcommand
+            ) + list(
+                self.expose_ports + self.volumes + self.environment
+            ) + self.labels + [
```

Comment:
is this OK?

Reply:
Is OK to resolve this, but IMHO is bad append list like that. You are using a weir effect of the list constructor, where `list([1] + [2]) == [1] + [2] == [1, 2]` only to avoid a flake8 warning.

```python
        _cmd = ['umoci', 'config'] + \
            self.maintainer + \
            self.user + \
            self.workingdir + \
            self.entry_command + \
            self.entry_subcommand + \
            self.expose_ports + \
            self.volumes + \
            self.environment + \
            self.labels + [
                '--image', container_name,
                '--created', datetime.utcnow().strftime(
                    '%Y-%m-%dT%H:%M:%S+00:00'
                )]
        Command.run(_cmd)
```

I am sorry, this is my last comment about the topic.